src/hold-item/domain/persistence/hold-item.domain.mapper.ts

```
@@ -0,0 +1,22 @@
+import { Injectable } from "@nestjs/common";
+import { HoldItemEntity } from "./hold-item.entity";
+import { promises } from "dns";
+import { HoldItemDTO } from "src/common/data/hold-item/hold-item.dto";
+
+@Injectable()
+export class HoldItemDomainMapper {
+    constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)